nevis/_util.py

```diff
@@ -3,6 +3,8 @@
 Provides utility methods (i.e. methods not directly related to ``nevis``).
 """
 import timeit
+from ._nevis_version import __version__
+from ._bng import Coords, Hill
```

Comment:
We're importing these into the `nevis` module in `__init__.py`. This means that we don't need to remember which file exactly they're in. So instead of doing this, just use `import nevis` and then later write `nevis.Coords` etc.

```suggestion

import nevis
```

Reply:
I see. Thanks for your detailed explanation!